rtc/introduction.md

```diff
@@ -0,0 +1,78 @@
+
+## End-to-end encryption
+
+The RTC plugin encrypts all content specific traffic that means the {{site.productname}} cloud services will not be able to read your data or be able to know who is editing all that is local to the current page running the plugin.
+
+### RTC enabled features
+
+Currently RTC only supports a subset of the features that {{site.productname}} has however we are working on bringing over more and more of those features.
+
+Here is a list of plugins we currently support:
+
+* [advlist]({{site.baseurl}}/plugins/opensource/advlist)
+* [autoresize]({{site.baseurl}}/plugins/opensource/autoresize)
+* [charmap]({{site.baseurl}}/plugins/opensource/charmap)
+* [emoticons]({{site.baseurl}}/plugins/opensource/emoticons)
+* [help]({{site.baseurl}}/plugins/opensource/help)
+* [hr]({{site.baseurl}}/plugins/opensource/hr)
+* [image]({{site.baseurl}}/plugins/opensource/image) (captioned images are not yet supported)
+* [insertdatetime]({{site.baseurl}}/plugins/opensource/insertdatetime)
+* [link]({{site.baseurl}}/plugins/opensource/link)
+* [lists]({{site.baseurl}}/plugins/opensource/lists)
+* [moxiemanager]({{site.baseurl}}/plugins/premium/moxiemanager)
+* [powerpaste]({{site.baseurl}}/plugins/premium/powerpaste)
+* [print]({{site.baseurl}}/plugins/opensource/print)
+* [save]({{site.baseurl}}/plugins/opensource/save)
+* [tabfocus]({{site.baseurl}}/plugins/opensource/tabfocus)
+* [tinydrive]({{site.baseurl}}/plugins/premium/tinydrive)
+* [visualblocks]({{site.baseurl}}/plugins/opensource/visualblocks)
+* [wordcount]({{site.baseurl}}/plugins/opensource/wordcount)
+
+### Settings that are not available when RTC is enabled
+
+* [custom_elements]({{site.baseurl}}/configure/content-filtering/#custom_elements)
+* [end_container_on_empty_block]({{site.baseurl}}/configure/advanced-editing-behavior/#end_container_on_empty_block)
+* [forced_root_block]({{site.baseurl}}/configure/content-filtering/#forced_root_block)
+* [forced_root_block_attrs]({{site.baseurl}}/configure/content-filtering/#forced_root_block_attrs)
+* [image_caption]({{site.baseurl}}/plugins/opensource/image/#image_caption)
+* [indent_use_margin]({{site.baseurl}}/configure/content-formatting/#indent_use_margin)
+* [keep_styles]({{site.baseurl}}/configure/content-filtering/#keep_styles)
+* [resize_img_proportional]({{site.baseurl}}/configure/advanced-editing-behavior/#resize_img_proportional)
+* [valid_children]({{site.baseurl}}/configure/content-filtering/#valid_children)
+
+### Browser support
+
+We only support modern desktop browsers Chrome, Firefox, Microsoft Edge and Safari.
```

Comment:
```suggestion
RTC only supports the latest versions of Chrome, Firefox, Microsoft Edge and Safari.
```
Trying to avoid "we" here again as personification shouldn't be used in technical documentation. I'm not sure if `desktop` needs to be here though. If it's only meant to be desktop and not mobile, we should add another sentence.